lib/src/generator/templates.runtime_renderers.dart

```diff
@@ -16380,6 +16380,8 @@ const _invisibleGetters = {
     'runtimeType'
   },
   'ClassElement': {
+    'augmentation',
```

Comment:
How is it used? These methods are marked `@experimental`, meaning that we would not like clients to use them yet.

Reply:
See the [README for mustachio](https://github.com/dart-lang/dartdoc/tree/main/tool/mustachio#map-of-properties). Every property of important classes is enumerated here; it is not hand-picked properties, but every property.

We can update the generator to exclude `@experimental` properties.

Reply:
OK, now I understand that these are generated.
I think we should not using experimental properties outside of the analyzer.

Reply:
What I worry is that if we change these getters in the analyzer, this will break `dartdoc`.
If this is not an issue, we can land this of course.

Reply:
Oh sorry, I missed where this is used. This is in the list of "invisible getters", so they cannot be used in the runtime renderers. An error is reported if an attempt is made to access this field.